data/db/build.gradle

```diff
@@ -62,6 +63,9 @@ kotlin {
             implementation Dep.Dagger.hiltAndroid
             implementation "androidx.datastore:datastore-preferences:1.0.0-alpha04"
 
+            // SQLDelight
```

Comment:
📝 
https://cashapp.github.io/sqldelight/multiplatform_sqlite/
  // The drivers needed will change depending on what platforms you target: